imuq:
  buffer_size: 20
  imu_rate: 100.0
  acc_noise: 0.005
  acc_bias_noise: 0.0005
  acc_bias_std: 0.005
  gyr_noise: 0.0005
  gyr_bias_noise: 0.00005
  gyr_bias_std: 0.0005
traj:
  integrate_acc: false
  update_acc_bias: false
grid:
  cell_rows: 2
  cell_cols: 8
  max_curve: 0.01
  max_var: 0.01
  nms: true
gicp:
  outer: 2
  inner: 3
  half_rows: 2
  half_cols: 6
  cov_lambda: 0.0
  min_eigval: 0.0
  imu_weight: 1.0
pano:
  rows: 256
  cols: 1024
  vfov: 90.0
  max_cnt: 20
  min_range: 0.5
  range_ratio: 0.1
  align_gravity: true
rigid: true
